layer/runtime_layer.cc

```diff
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "runtime_layer_data.h"
-
 #include <cassert>
 #include <cstdint>
 #include <cstring>
 #include <functional>
 #include <string>
 
+#include "runtime_layer_data.h"
```

Comment:
What does this include bring?  It does not seem to be used.

Reply:
Clang-format reordered includes. `RuntimeLayerData` is used in almost every function.